css/style.css

```diff
@@ -0,0 +1,393 @@
+@font-face {
+  font-family: "Lato";
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: url("../fonts/Lato-Light.ttf") format("truetype");
+}
+@font-face {
+  font-family: "Lato";
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+  src: url("../fonts/Lato-Regular.ttf") format("truetype");
+}
+@font-face {
+  font-family: "Lato";
+  font-style: normal;
+  font-weight: 700;
+  font-display: swap;
+  src: url("../fonts/Lato-Bold.ttf") format("truetype");
+}
+@font-face {
+  font-family: "Lato";
+  font-style: normal;
+  font-weight: 900;
+  font-display: swap;
+  src: url("../fonts/Lato-Black.ttf") format("truetype");
+}
+* {
+  margin: 0;
+  padding: 0;
+}
+
+ol, ul {
+  list-style: none;
+}
+
+a {
+  -webkit-text-decoration-line: none;
+          text-decoration-line: none;
+}
+
+* {
+  font-family: "Lato", sans-serif;
+}
+
+.wrapper {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  max-width: 1600px;
+  width: 100%;
+}
+
+.header {
+  background-color: #222;
+  min-height: 72px;
+}
+
+.header__container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 18px;
+  padding-left: 230px;
+  padding-right: 230px;
+}
+
+.header__button {
+  background-color: #222;
+  border: none;
+}
+
+.header__menu-list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  padding-left: 440px;
+}
+
+.header__menu-item {
+  margin-right: 33px;
+}
```

Comment:
.header__menu-item :not(:last-of-type) {
    margin-left: 33px;
}

Reply:
done